src/test/java/core/basesyntax/files/CustomFileWriterTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.files;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CustomFileWriterTest {
+    private static final String FILE_TO_WRITE = "src/main/resources/finalReport.csv";
```

Comment:
```suggestion
    private static final String FILE_TO_WRITE = "src/test/resources/finalReport.csv";
```
You should put your test resources in the test folder